toolkit/tools/imagecustomizer/docs/building-packages.md

```diff
@@ -0,0 +1,85 @@
+# Building custom packages
+
+This is a guide on how to build your own packages for Azure Linux.
+
+This guide uses an Azure Linux container to build the packages.
+This method is generally a lot easier than trying to use the Azure Linux toolkit, which
+is only really necessary when you need to build Azure Linux from scratch.
+
+## Quick start
+
+1. Create a file called `samplescript.sh` with the following contents:
+
+   ```bash
+   echo "Hello, World"
+   ```
+
+2. Create a file called `samplepackage.spec` with the following contents:
+
+   ```rpmspec
+   Summary:        A sample package
+   Name:           samplepackage
+   Version:        0.0.1
+   Release:        1%{?dist}
+   Vendor:         Contoso
+   License:        Proprietary
```

Comment:
might be illustrative to add a `BuildRequires: words` or something.

Reply:
`BuildRequires` is an odd one, because it doesn't really do anything useful in this situation. Any build dependencies still have to be added to the `Dockerfile` regardless of if they are specified in the `.spec` file.

Reply:
Though, I did add a `Requires:` since those are likely useful to users.